lava-provider/example_rpcprovider.yml

```diff
@@ -0,0 +1,27 @@
+endpoints:
+  - api-interface: tendermintrpc
+    chain-id: LAV1
+    network-address:
+        address: "0.0.0.0:2001"
+    node-urls:
+      - url: http://provider.hurricane.akash.pub:<26657>
+      - url: ws://provider.hurricane.akash.pub:<26657>/websocket
+  - api-interface: grpc
+    chain-id: LAV1
+    network-address:
+        address: "0.0.0.0:2001"
+    node-urls:
+      - url: provider.hurricane.akash.pub:<9090>
+  - api-interface: rest
+    chain-id: LAV1
+    network-address:
+        address: "0.0.0.0:2001"
+    node-urls:
+      - url: http://provider.hurricane.akash.pub:<1317>
+
+#For ETH CHAIN
+  - api-interface: jsonrpc
+    chain-id: ETH1
+    network-address: 127.0.0.1:2221
+    node-urls:
+      - url: wss://eth-rpc/ws
```

Comment:
just like you put `eth-rpc` here, how about replacing:
- `http://provider.hurricane.akash.pub:<1317>` with `http://rest-node:1317`
- `provider.hurricane.akash.pub:<9090>` with `grpc-node:9090`
- `http://provider.hurricane.akash.pub:<26657>` with `http://rpc-node:26657`

as well as change `wss://` to `ws://`  since the general example would be for non-TLS endpoints. ( you already have a `ws://` for the RPC ;-))

You can also add an extra example config where all would be TLS protected, e.g.:

- **rest** (aka API, aka LCD): `https://rest-node:443`
- **gRPC**: `grpcs://grpc-node:9090`  (I think it should be `grpcs://` but not sure whether `lavad rpcprovider` reads `grpcs://` (tls) vs `grpc://` (no-tls); you'll need to check it in the docs or test / ask the Lava guys)
- **rpc**: `https://rpc-node:443`